packages/material-ui/src/styles/experimentalStyled.js

```diff
@@ -85,7 +86,12 @@ const experimentalStyled = (tag, options, muiOptions = {}) => {
       });
     }
 
-    return defaultStyledResolver(...stylesWithDefaultTheme);
+    stylesWithDefaultTheme.push((props) => {
+      const theme = isEmpty(props.theme) ? defaultTheme : props.theme;
+      return styleFunctionSx(props.sx, theme);
+    });
+
+    return defaultStyledResolver(stylesWithDefaultTheme);
```

Comment:
Necessary for fixing `You have illegal escape sequence in your template literal` Spreading is dropped. This will be revisit separately after the system is implemented, but this should help at least with eliminating the warning.